narayana-release-process.sh

```diff
@@ -165,13 +165,14 @@ else
 fi
 
 rm -rf $PWD/localm2repo
-./build.sh clean install -Dmaven.repo.local=${PWD}/localm2repo -DskipTests -gs ~/.m2/settings.xml -Dorson.jar.location=$ORSON_PATH -Pcommunity
+# It is important in the deploy step that if you are deploying to nexus you provide a reference to your settings file as the ./build.sh overrides the default settings file discovery of Maven. Please see https://github.com/jbosstm/narayana/wiki/Narayana-Release-Process for details of the settings.xml requirements
+./build.sh clean install -Dmaven.repo.local=${PWD}/localm2repo -DskipTests -Dorson.jar.location=$ORSON_PATH -Pcommunity
 if [[ $? != 0 ]]
 then
   echo 1>&2 Could not install narayana
   exit
 fi
-./build.sh clean deploy -Dmaven.repo.local=${PWD}/localm2repo -DskipTests -gs ~/.m2/settings.xml -Dorson.jar.location=$ORSON_PATH -Prelease,community -DskipNexusStagingDeployMojo=false
```

Comment:
I don't think this "-gs ~/.m2/settings.xml" can be removed for the reasons in the comment above :)

Reply:
My apologies, my mistake. I will restore '-gs ~/.m2/settings.xml' during the install and the deploy executions.

Reply:
Sorry, I was not clear. It should stay on the `deploy` step but be removed from the `install` step (unless you know why it needs to be there on the `install` step?)

Reply:
I see your point, I agree there is need to have it in the install phase. +1

Reply:
Thank you!